dynawo/sources/Modeler/ModelManager/DYNModelManager.cpp

```diff
@@ -557,8 +557,13 @@ ModelManager::dumpParameters(map< string, string > & mapParameters) {
   // same method can't be applied to string due to the implicit cast from
   // modelica_string to string
   vector<string> paramsString;
-  for (unsigned int i = 0; i < modelData()->nParametersString; ++i)
+  for (unsigned int i = 0; i < modelData()->nParametersString; ++i) {
     paramsString.push_back(simulationInfo()->stringParameter[i]);
+  }
+  std::vector<std::string> delays = delayManager_.dumpDelays();
+  for (std::vector<std::string>::const_iterator it = delays.cbegin(); it != delays.cend(); ++it) {
+    paramsString.push_back(*it);
```

Comment:
Please create a dedicated vector instead of reusing another

Reply:
We'd have to change the whole parsing system, as the dump file ends with string parameters that cannot be parsed individually. All string parameters must be in the same vector at the beginning

Reply:
OK